config/ytt/static.yaml

```diff
@@ -2,12 +2,14 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
Please don't change this file for now, we have a known issue when run `make manifest` command after @gwang550 's webhook PR merged, need to fix it in future.